pycbc/inference/models/marginalized_gaussian_noise.py

```diff
@@ -371,11 +371,9 @@ def _setup_prior(self):
                                    for i in self._dist_array])
         if self._margtime:
             bounds = self._marg_prior["time"].bounds
-            self._time_array = numpy.linspace(bounds["time"].min,
-                                              bounds["time"].min, 10**4)
-            self.time_prior = numpy.array(
-                                  [self._marg_prior["time"].pdf(time=i) for
-                                   i in self._time_array])
```

Comment:
Why did you remove the time prior bit? I think the idea with this was so you could use a non-uniform prior on the time. It looks like there was a bug there though in that `time_array` should've been between the min/max bounds not min/min.